README.md

```diff
@@ -978,11 +978,17 @@ let app = express(),
 
 app.post('/users', (req, res, next) => {
   const user = new User(req.body);
-  user.save(err => {
+  user.save(async err => {
```

Comment:
@mmiller42 : Can we replace the callback based example of `save` method with async/await? The middleware for '/users' would then become an async method. And we would have a single try/catch to handle errors from both `user.save` and `agenda.now`. Does that make sense?